helm/kubeedge-cloudcore/values.yaml

```diff
@@ -40,8 +40,7 @@ cloudCore:
     cloudHub:
       # Caution!: Leave this entry to empty will cause CloudCore to exit abnormally once KubeEdge is enabled.
       # At least a public IP Address or an IP which can be accessed by edge nodes must be provided!
-      advertiseAddress:
-        - ""
+      advertiseAddress: []
```

Comment:
This is now an empty array as the user doesn't need to supply any IPs - the hook job added in this PR will take care of getting the IP and updating the configmap.